tools/packaging/static-build/cache_components.sh

```diff
@@ -70,7 +90,17 @@ Usage: $0 "[options]"
 	Options:
 		-c	Cloud hypervisor cache
 		-k	Kernel cache
+			* Can receive a TEE environnment variable value, valid values are:
+			  * tdx
+			  If no TEE environment is passed, QEMU is built without TEE support.
```

Comment:
```suggestion
			  If no TEE environment is passed, the kernel is built without TEE support.
```
I think this should be referencing the kernel, rather than QEMU here?